### ISSUE TITLE
Cannot use string as body any more

### ISSUE THREAD

**Issue by t-animal** [mocks-server/main#363]
**Describe the bug**
The default-variant in 2.x allowed responding with a string. The json-variant in 3.5 only allows arrays and objects. A handler like
```
{
  id: "server-broken",
  handler: "json",
  response: {
    status: 500,
    body: "Internal server error" 
  }
}
```
doesn't work any more.

**To Reproduce**
Use response like specified above.

**Expected behavior**
The json route variant should support responding with strings directly.

**Comment by javierbrea** [comment]
Hi @t-animal,
This is intentional, because the "json" handler is intended to handle only json responses. It next major version it will add automatically a content-type header with "application/json" value.
I have planned to add another handler named "text" to handle string responses. Meanwhile, you can still use the deprecated "default" handler if you don't define an specific handler:

```js
{
  id: "server-broken",
  response: {
    status: 500,
    body: "Internal server error" 
  }
}
```

**Comment by t-animal** [comment]
I used a middleware now, sending the string manually:

```
  handler: "middleware",
  response: {
    middleware: (req, res, next) => res.status(400).send("Invalid request, need <property>")
  }
```

**Comment by javierbrea** [comment]
Related to #366

**Comment by javierbrea** [comment]
FYI @t-animal: The new "text" variant handler is available in [version 3.7.0](https://github.com/mocks-server/main/releases/tag/v3.7.0)
